bib/cmd/bootc-image-builder/image.go

```diff
@@ -498,9 +498,8 @@ func manifestForISO(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest, erro
 	img.Kickstart.OSTree = &kickstart.OSTree{
 		OSName: "default",
 	}
-	// use lorax-templates-rhel if the source distro is not Fedora with the exception of Fedora ELN
 	img.UseRHELLoraxTemplates =
-		c.SourceInfo.OSRelease.ID != "fedora" || c.SourceInfo.OSRelease.VersionID == "eln"
+		c.SourceInfo.OSRelease.ID == "rhel" || c.SourceInfo.OSRelease.ID == "centos" || c.SourceInfo.OSRelease.VersionID == "eln"
```

Comment:
Also needs `almalinux`.

Reply:
Yes, `ID_LIKE` having `rhel` is a better check here.

Reply:
Thanks @Conan-Kudo and @supakeen - I tweaks this to use ID_LIKE now.